scripts/generate

```diff
@@ -391,23 +406,30 @@ const generate =
 def ('generate')
     ({})
     ([$.String, Html.Type])
-    (pipe ([replace (regex ('gm') ('^(#{1,6}) <span id="([^"]*)">❑ (.*)</span>$'))
-                    (($0, $1, $2, $3) => `<h${$1.length} id="${$2}">${$3}</h${$1.length}>`),
-            replace (regex ('gm') ('^#### <a name="(.*)" href="(.*)">`(.*)`</a>$'))
-                    (($0, $1, $2, $3) => Html.unwrap (headingToMarkup ($1) ($2) ($3))),
-            replace (regex ('gm') ('^```javascript\\n(> [^]*?)^```\\n'))
-                    (($0, $1) => Html.unwrap (doctestsToMarkup ($1))),
+    (pipe ([replace (contramap (justs) (([$1, $2, $3]) => `<h${$1.length} id="${$2}">${$3}</h${$1.length}>`))
```

Comment:
Curious about why you're using `contramap` in these cases, vs `compose` or `pipe`.

Reply:
I began using `contramap` in #87:

```javascript
//    replace__ :: RegExp -> (Array String -> String) -> String -> String
const replace__ = flip (compose (flip (replace_)) (contramap (justs)));
```

I prefer `contramap` to `compose` and `map` in this context because it takes the common input first, meaning that the occurrences of `(justs)` are aligned.

I prefer `contramap` to `pipe` in this context because `contramap (justs)` is a complete expression. As such, it could be given a name or used in a composition.